ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -51,7 +51,9 @@ internal fun Template1(state: PaywallViewState.Loaded, viewModel: PaywallViewMod
         modifier = Modifier.fillMaxSize(),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Template1MainContent(state)
+        Column(modifier = Modifier.weight(1f)) {
+            Template1MainContent(state)
+        }
```

Comment:
This fixes an issue that happened in some devices, specially in landscape where the purchase button and footer weren't visible. By adding a weight to the main content, it has less priority than the button/footer so those will always show.

Reply:
Would adding the weight to the outer colum work too?

Reply:
No, if we apply the weight to the outer column, that means that it won't apply to the inner views. Weights work with siblings by allowing you to set a relative weight between them to use the available space. In case one or more of the siblings don't have a weight attribute, those will have priority over those that do have it, which is what I'm doing here (give priority to the button + footer over the main content to display on the screen)